src/site/markdown/server/manage-webadmin.md

```diff
@@ -3048,6 +3049,40 @@ Response codes:
  - 204: Success (No content)
  - 400: Invalid request
  - 404: The mail queue does not exist
+ 
+### RabbitMQ republishing a mail queue from cassandra
+
+```
+curl -XPOST http://ip:port/mailQueues/{mailQueueName}?action=RepublishNotProcessedMails&olderThan=1d
+```
+
+This method is specific to the distributed flavor of James, which relies on Cassandra and RabbitMQ.
+In case of a RabbitMQ crash resulting in a lost of messages. This task can be launched to repopulate the
```

Comment:
In case of a RabbitMQ crash resulting in a lost of messages, this task can be launched...